### PR TITLE
run.ver upgrades

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
Upgrading run.ver to have the following upgrades:

evs_ver to v2.0
bufr_ver to v12.0.0 for bufr splitting (already in PR #477)
obsproc_ver to v1.2
upgrading met_ver to 12.0.0-beta4
upgrading metplus_ver to 6.0.0-beta4
adding metplotpy_ver, metcalcpy_ver, and metdataio_ver all to 3.0.0-beta4

Also upgrading ecf and dev drivers that were changed for the obsproc upgrade for increased timing.

No testing was done.  All testing is being done in the emc.vpppg parallel and other PRs.

This can be merged without any testing.

### CODE REVIEW

File: dev/drivers/scripts/stats/cam/jevs_cam_hrrr_grid2obs_stats.sh
Comment (AliciaBentley-NOAA) [inline dev/drivers/scripts/stats/cam/jevs_cam_hrrr_grid2obs_stats.sh]: @PerryShafran-NOAA `#PBS -A VERF-DEV` seems to appear twice (on lines 6 and 7). Can line 7 be removed?

File: dev/drivers/scripts/stats/mesoscale/jevs_mesoscale_rap_grid2obs_stats.sh
Comment (AliciaBentley-NOAA) [inline dev/drivers/scripts/stats/mesoscale/jevs_mesoscale_rap_grid2obs_stats.sh]: @PerryShafran-NOAA Can you please add the space back before "export" on line 40 so that the only change in this script is the walltime?

**Comment by AliciaBentley-NOAA** [comment]
@PerryShafran-NOAA @ShelleyMelchior-NOAA I've looked over this PR and I think it looks good! I made two small comments above. I recommend that we wait to merge this PR until after Shelley returns and EVS v1.0.9 becomes operational on Monday. I'd like us to do a "diff" of the ops ecf/ scripts and this PR's ecf/ scripts before this PR is merged, to make sure we caught everything in ops EVS v1.0.9. Thanks!

File: dev/drivers/scripts/stats/mesoscale/jevs_mesoscale_rap_grid2obs_stats.sh
Comment (PerryShafran-NOAA) [reply dev/drivers/scripts/stats/mesoscale/jevs_mesoscale_rap_grid2obs_stats.sh]: Done.

File: dev/drivers/scripts/stats/cam/jevs_cam_hrrr_grid2obs_stats.sh
Comment (PerryShafran-NOAA) [reply dev/drivers/scripts/stats/cam/jevs_cam_hrrr_grid2obs_stats.sh]: Fixed.

**Comment by AliciaBentley-NOAA** [comment]
@PerryShafran-NOAA One last thing! Please update machine.conf to include a new setting: LOG_TO_TERMINAL_ONLY = TRUE. Thanks! 

CC @ShelleyMelchior-NOAA

**Comment by ShelleyMelchior-NOAA** [comment]
Are the increased walltimes in the dev drivers and ecf scripts b/c of obsproc.v1.2 and increased prepbufr file sizes? I suppose this was to match NCO settings in prod, too, right?

**Comment by PerryShafran-NOAA** [comment]
@ShelleyMelchior-NOAA Yes, these were done to match the settings in prod.  The increased walltimes are indeed due to the larger prepbufr files in obsproc v1.2.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Changes look good. All changes have been running in the EVS para for many weeks.

**Comment by ShelleyMelchior-NOAA** [comment]
This PR is approved for merge into develop.

**Pull request merged**
